src/bounding.rs

```diff
@@ -5,7 +5,7 @@ use crate::Capability;
 use std::io::Error;
 
 pub fn clear() -> Result<(), CapsError> {
-    for c in super::all() {
+    for c in runtime::thread_all_supported() {
```

Comment:
I wonder if it makes more sense to do `has_cap(c).unwrap_or(false)` instead (and update `read()` in the same way too).
That arrangement should reduce the total number of syscalls that we perform for this.